scripts/scripts.js

```diff
@@ -354,6 +354,39 @@ function decorateH3(main) {
     });
   }
 }
+
+/**
+ * Checks for the same domain or not and if ends with pdf
+ */
+function isSameDomainOrPdf(url) {
+  // Create an anchor element to parse the URL
+  const ancElement = document.createElement('a');
+  ancElement.href = url;
+
+  // Check if the URL ends with ".pdf" or is from the same domain
+  return (
+    window.location.hostname === ancElement.hostname || ancElement.pathname.toLowerCase().endsWith('.pdf')
```

Comment:
I would add a few more conditions here for:

* ending with `.hlx.live`
* ending with `.hlx.page`
* ending with `servicenow.com` (in case we'll use some of your other domains for testing, like: https://www.webstg.servicenow.com/ , https://webqa.servicenow.com/, https://webdev.servicenow.com/)

oterhwise relative links on all these domains will open in a new tab.

Reply:
Added these conditions as well except servicenow.com. I believe the last one would not be needed. Let me know your thoughts on this.